common/persistence/execution_manager.go

```diff
@@ -764,6 +765,12 @@ func (m *executionManagerImpl) GetCurrentExecution(
 	request *GetCurrentExecutionRequest,
 ) (*GetCurrentExecutionResponse, error) {
 	response, respErr := m.persistence.GetCurrentExecution(ctx, request)
+
+	var notFound *serviceerror.NotFound
+	if errors.As(respErr, &notFound) {
+		// strip persistence-specific error message
+		respErr = serviceerror.NewNotFound("workflow not found")
+	}
```

Comment:
Initially, I had a fix in the persistence-specific implementations; but this seems easier and safer (as it covers any current and future ones).

Reply:
For the record, SQL was returning `sql: no rows in result set` and Cassandra returned `operation GetCurrentExecution encountered not found`.